src/main/java/uk/gov/hmcts/reform/blobrouter/data/envelopes/EnvelopeRepository.java

```diff
@@ -136,6 +136,15 @@ public int markAsDeleted(UUID id) {
         );
     }
 
+    public int updatePendingNotification(UUID id) {
```

Comment:
doesn't say what it updates it to? Maybe `markAsPendingNotification`?

Reply:
added another parameter for the value, because it is used in 2 places now (when the envelope is rejected and notification sent)